models/role.js

```diff
@@ -0,0 +1,12 @@
+import { Schema, model } from 'mongoose'
+import { ensureFieldUniquity } from "./utils";
+
+const RoleSchema = new Schema({
+    name: { type: String, required: true, unique: true },
+    color: { type: String, required: true },
```

Comment:
It should be validate

Reply:
A moment ago I have added validation in Joi schema. But now I realized that we will save a few bytes on each document by storing the color as an integer, since it's always hexadecimal

Reply:
Which one format do you suggest to store colors?

Reply:
An integer representing a hexadecimal value